### PR TITLE
Add loading spinner when the user logs in or out

### LINKED ISSUE #22: Add Logout view
### Description
- We need to show a message when the user logout from trello clone.

### Proposal
- IDK

### Links / references
image

### PR DESCRIPTION
**Pull request by albertoescala**
### Description
Add loading spinner when the user logs in or out.

This PR:
 - Add a spinner
 - Remove logout component

⚠️ Please review the functionality before approving it. It has breaking changes. ⚠️ 

Closes #22 

🔥 Thank you!

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ class Home extends Component {
   logout = () => {
     this.props.resetMenuState()
     auth.logout()
+    auth.login()
```
Comment (nickeyvee) [inline src/components/Home/Home.js]: I'd make a comment here saying that this redirects the user to the auth0 login form on logout.

File: README.md [hunk 2]
```diff
@@ -34,8 +34,6 @@ This is a list of proposed features to add after MVP is completed.
...
 
 ### Authors
-@albertoescala
-
-@Debasree Samadder
-
-@Nicholas Vorraso
+* [Alberto Escala](https://github.com/albertoescala)
+* [Nicholas Vorraso](https://github.com/nickeyvee)
+* [Debasree Samadder](https://github.com/debasree888)
```
Comment (nickeyvee) [inline README.md:39]: Thanks for doing this, never got to it.

**Review by nickeyvee — CHANGES_REQUESTED** [review]
See line 41 in `Home.js`

**Comment by albertoescala** [comment]
@nickeyvee I already added.

**Review by nickeyvee — APPROVED** [review]

**Pull request merged**